shibata revised for #37, 38, 39,40 (#42)
fujitake/vantiq-related: vantiq-apps-development/1-day-workshop/docs/eng/4-Lab03_Sources.md

3. Set the Source Type to _MQTT_.

4. Open "Server URI" tab, click the _+ Add Server URI_ and set the [URI of the MQTT Broker which you prepared by yourself](./2-Lab01_Preparation.md#mqtt_broker_setting).  

5. Open "Topic" tab, click the _+ Add Topic_ and type the following Topic in the Topic field.

Topic： "_/***your name***/pump/Temp_"

> ＊ **Type the same value that was entered in "TempSensorTopic"** at [Lab 01 - Stap 5 (Configure the Data Generator)](./2-Lab01_Preparation.md#data_generator_setting). Also, the Topic name should **not contain double quotation marks**. Please do **not include spaces before or after** the Topic name.

Reviewer: ```suggestion
＊ **Type the same value that was entered in "TempSensorTopic"** at [Lab 01 - Stap 5 (Configure the Data Generator)](./2-Lab01_Preparation.md#data_generator_setting). Also, the Topic name should **not include enclosing double quotations nor extra spaces**. 
```
